Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+
+Note: Keep in mind that at the time of writing this document, we’ve made our best effort 
+ to survey the container registry landscape based on publicly available materials.
+ If you find something outdated (with respect to the time of writing this document) or
+ outright erroneous, please submit a PR or raise an issue, and we’ll fix it right away.
+
+## OSS health check
+
+This section evaluates the health of the open-source projects that were selected from 
+the currently available solutions. SCS-compliant software must fulfill all OSS health 
+checks defined by the [OSS-Health](https://github.com/SovereignCloudStack/standards/blob/main/Design-Docs/OSS-Health.md) 
+document. The main health checks are:
+- Four Opens (code is fully open source, community is open and diverse, development 
+  process is open, design process is open)
+- Maturity
+- Security
+- Activity
+- Lock-in risk assessment
+
+Each selected OSS project is evaluated based on the above checks, and it is classified 
+to one of three categories as follows:
+
+- :heavy_check_mark: The project passed all OSS health checks and will be considered 
+  further as a valid candidate.
+
+- :grey_question: The project passed almost all OSS health checks. 
+  There is a place for improvement, but the missing points are not crucial from the OSS 
+  health check perspective. The project will be considered further as a valid candidate.
+
+- :x: The project does not pass the OSS health checks. Some OSS health check 
+  showstoppers have been found (e.g. open core software, not actively maintained). 
+  The project is filtered at this stage and won't be considered further.
+
+Refer to the list of evaluated projects with their classified categories and comments.
+
+- :heavy_check_mark: [Harbor][harbor]
+  - Harbor project meets all "four opens"
+    - Source code is open and available under the [Apache 2 license](https://github.com/goharbor/harbor/blob/main/LICENSE)
+    - Community is [open](https://github.com/goharbor/harbor#community, structured and 
+      well organized via [workgroups](https://github.com/goharbor/community) and 
+      various communications channels e.g. Slack, mailing lists, etc. 
+      (#harbor Slack channel contains 3k+ members)
+    - The development process is open via GitHub issues and well described in the 
+      [contributing](https://github.com/goharbor/harbor/blob/main/CONTRIBUTING.md) 
+      document
+    - The design process is open via GitHub issues. Proposals are [public](https://github.com/goharbor/community/tree/main/proposals).
+      The decision process is well described as well. The project's roadmap is 
+      available in the [roadmap](https://github.com/goharbor/harbor/blob/main/ROADMAP.md) document
+  - Maturity is on the CNCF [graduation](https://www.cncf.io/projects/harbor/) level.
+    CNCF graduated project is considered stable, widely adopted and production-ready
```

Comment:
Replace `CNCF graduated project is considered stable, widely adopted and production-ready` with either:

- `A CNCF graduated project is considered to be stable, widely adopted and production-ready`
- `CNCF graduated projects are considered to be stable, widely adopted and production-ready`

My preference would be for the latter.